### ISSUE TITLE
CFCFGBuilder and CFGBuilder duplicate logic for enhanced for loops

### ISSUE THREAD

**Issue by GoogleCodeExporter** [typetools/checker-framework#356]
```
CFCFGBuilder and CFGBuilder duplicate the logic to create a CFG for enhanced 
for loops in visitEnhancedForLoop.
```

Original issue reported on code.google.com by `weitz...@cs.washington.edu` on 10 Sep 2014 at 11:53

**Comment by smillst** [comment]
@wmdietl  
Werner, I think you fixed this issue in commit 242902ce67bf008417f2cc3c6228a69c03683157.   Can you confirm and close this issue?
Thanks!

**Comment by wmdietl** [comment]
Yes, looks like this is fixed.
